SpecflowPages/Utils/Start.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Onboarding_Task_Mars_Specflow.SpecflowPages.Helpers;
+using Onboarding_Task_Mars_Specflow.SpecflowPages.Pages;
+using TechTalk.SpecFlow;
+
+namespace Onboarding_Task_Mars_Specflow.SpecflowPages.Utils
+{
+    [Binding]
+    public class Start : DriverHelpers
```

Comment:
this is the hooks class i would avoid extend to DriverHelpers instead create an object and then call the initialise method like you have Initialize()